### ISSUE TITLE
bibtex(8) man section? disabling?

### ISSUE THREAD

**Issue by kberry** [plk/biblatex#1239]
In the warning
```
Package biblatex Warning: Using fall-back BibTeX(8) backend:
```

Is the (8) a Unix man page reference? Sorry, I can't imagine what else it would refer to. The bibtex man page has never been in section 8 (not a sysadmin tool), always section 1. Nor is the man page named "BibTeX", it is "bibtex", so BibTeX(1) would still be wrong, and bibtex(1) would probably confuse people. And besides, the main documentation is not the man page. So maybe just delete the "(8)"?

Also, it would be nice if there were a way to disable this warning. (If there already is, great, please let me know. I couldn't find it.) I run an article I'm working on a hundred times, I don't need to see this every time; I know I'm using bibtex even though you don't want me to :). --thanks, karl.

**Comment by plk** [comment]
Assuming @moewew has no objection, I have removed the warning completely for 3.19 as `biber` has been the default for a long time now and we can assume anyone explicitly using `backend=bibtex` knows they typed that ... that manual section thing must be very old and clearly is wrong anyway since it's displayed on all systems, non-unix included. 3.19 is in the SF "development" folder.

**Comment by moewew** [comment]
I think "BibTeX(8)" is a compact way to say "BibTeX or BibTeX8". I don't think this is supposed to refer to any man pages.

There is almost no code on the `biblatex` end that distinguishes between BibTeX and BibTeX8, but formally the `backend` option can take the values `biber`, `bibtex` and `bibtex8`. So it makes sense that the warning would want to mention both BibTeX and BibTeX8 explicitly.

---

Personally I always liked the warning. It was always the first authoritative thing I could quote in a forum post to persuade people to change to Biber. And maybe it help people who read warnings, but for some reason have to work with bad templates.

In theory I think it would be better not to have this general warning, but instead have all commands/features that require Biber issue specific warnings with `backend=bibtex,`, but that requires quite a lot of work for things that we don't want top encourage anyway (namely BibTeX use).

**Comment by plk** [comment]
Ah, good point - that's clearly a bibtex/bibtex8 reference - coming from a UNIX background it really does look like a manual page section reference. We could put an option in to disable the warnings but it seems like a somewhat gratuitous use of an option for such a specific issue. I am somewhat ambivalent about the warning as not a lot of people look at these anyway - styles requiring bibtex should be saying that in their docs I suppose.

**Comment by kberry** [comment]
I didn't mean to suggest doing anything that is a nontrivial amount of work.

About the wording, I suggest simply "bibtex or bibtex8" to avoid all
issues about interpretation as man pages. Since your option names are
lowercase. And I don't think I've ever seen bibtex8 referred to as
"BibTeX8". That's part of why I thought it was a man page in the first place.

About disabling, well, the point is the same for all warnings: once it
has been read, and the user needs to proceed anyway, it becomes noise,
adding to the general problem of (La)TeX noise obscuring real
problems. Sure, it's hardly a showstopper, and I can filter it out in
other ways, but un-silenceable warnings in general seem like a bug to
me.

But if you don't think it's worth the trouble to create and document, I
understand. I won't argue farther. --thanks, karl.

**Comment by plk** [comment]
@moewew - How about, since biblatex warnings are all wrapped in various warn/error macros, we add a general flag to not print warnings to the terminal? I'm assuming that Karl is referring to the terminal warnings, not the log? I can see some use in that for people with a more automated flow who are only looking in logs for errors/warnings.

**Comment by moewew** [comment]
Hmm, I don't think I'd like to add an option to disable all warnings (I'm too afraid templates set this option to get rid of warnings). For @kberry's use case that doesn't seem to be that useful, either, since I'm assuming he'd want to be warned about other issues with `biblatex`, just not the backend.

**Comment by moewew** [comment]
There is precedent for commands to disable certain warnings

https://github.com/plk/biblatex/blob/442473354b79126a2fb5ec89dd692dff0c4ef384/tex/latex/biblatex/biblatex.sty#L16345-L16348

so we could add one for the backend.

**Comment by kberry** [comment]
You guys are spending too much time on this :). At this point, I suggest just changing the wording to "bibtex or bibtex8" to avoid the seeming man page and let it go at that.  Anyone who cares about filtering TeX output won't have trouble dropping that warning too. Thanks.

**Comment by plk** [comment]
Let's just do that for now then. I'm not sure we need to say "or bibtex8" as "bibtex" in basically all contexts just means "whatever bibtex you use".

**Comment by kberry** [comment]
Your point that the backend identifier can be "bibtex" or "bibtex8" makes
me think the message would be clearest if it mentioned both. --best, karl.

**Comment by plk** [comment]
Fixed in 3.19 release
